journals/journal_abbreviations_general.txt

```diff
@@ -3724,10 +3724,11 @@ Proceedings of the Physical Society of London = Proc. Phys. Soc. London
 Proceedings of the Physical Society. Section A = Proc. Phys. Soc. London, Sect. A
 Proceedings of the Physical Society. Section B = Proc. Phys. Soc. London, Sect. B
 Proceedings of the Royal Irish Academy = Proc. R. Ir. Acad.
-Proceedings of the Royal Society of London, Series A = Proc. R. Soc. London, Ser. A
-Proceedings of the Royal Society of London, Series A: Mathematical and Physical Sciences = Proc. R. Soc. London, Ser. A
-Proceedings of the Royal Society of London, Series B = Proc. R. Soc. London, Ser. B
-Proceedings of the Royal Society of London, Series B: Biological Sciences = Proc. R. Soc. London, Ser. B
+Proceedings of the Royal Society of London. Series A = Proc. R. Soc. London, Ser. A.
+Proceedings of the Royal Society of London. Series A - Mathematical and Physical Sciences = Proc. R. Soc. London, Ser. A Math Phys. Sci.
+Proceedings of the Royal Society of London. Series B = Proc. R. Soc. London, Ser. B
+Proceedings of the Royal Society of London. Series B: Biological Sciences = Proc. R. Soc. London, Ser. B
+Proceedings of the Royal Society of London. Series A, Containing Papers of a Mathematical and Physical Character = Proc. R. Soc. London, Ser. A.
```

Comment:
wasn't able to locate the correct abbreviation for this journal (PRSA has renamed the journal several times in it's history)

Reply:
Thanks nonetheless for your contribution!